packages/loader/src/gltf/parser/GLTFSkinParser.ts

```
@@ -39,7 +39,7 @@ export class GLTFSkinParser extends GLTFParser {
         const rootBone = entities[skeleton];
         skin.rootBone = rootBone;
       } else {
-        const rootBone = this._findSkeletonRootBone(joints, entities);
+        const rootBone = this._findSceneRootBone(context, joints, entities) ?? this._findSkeletonRootBone(joints, entities);
```

ESLint

[error] Insert `⏎·········`

(prettier/prettier)